docs/go-quickstart.md

```diff
@@ -0,0 +1,168 @@
+# Quickstart: Using Thema in Go
+
+The tutorial demonstrates how a user can create an empty Lineage using the thema CLI, define a simple schema and use it to generate Go types and bindings, and use the bindings to validate data against a selected schema in the Lineage.
+
+## Defining a schema
+The most fundamental task in thema is writing a lineage. We can use the Thema CLI to generate an empty lineage using the below command:
+
+```
+thema lineage init empty --name ship > ship.cue 
+```
+
+This command will genearte an empty lineage named `ship` and store the output in a file named `ship.cue`. The genearted lineage looks like below
+
+```cue
+package ship
+
+import "github.com/grafana/thema"
+
+thema.#Lineage
+name: "ship"
+seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				// TODO (delete me - first schema goes here!)
+			},
+		]
+	},
+]
+```
+
+Let's define a simple schema as an object containing two fields named `firstField` of type `string` and `secondField` of type `int`,  
+
+```cue
+package ship
+
+import "github.com/grafana/thema"
+
+thema.#Lineage
+name: "ship"
+seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				firstField: string
```

Comment:
Let's definitely include some field comments right from the start, so that the user immediately has the impression that documenting their fields is important.

Reply:
Yup added comments as you mentioned in the ship example